pkg/core/mpt/billet.go

```diff
@@ -191,7 +191,7 @@ func (b *Billet) incrementRefAndStore(h util.Uint256, bs []byte) {
 		}
 		cnt++
 		if len(data) == 0 {
-			data = append(bs, 0, 0, 0, 0)
+			data = append(bs, 1, 0, 0, 0, 0)
```

Comment:
`(b *Billet) GetFromStore(h util.Uint256)` should also be adjusted (line 341) wrt this extra byte.